packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -70,6 +70,12 @@ function get_base_diffusers_model(model: ModelData): string {
 	return model.cardData?.base_model?.toString() ?? "fill-in-base-model";
 }
 
+function get_prompt_from_diffusers_model(model: ModelData): string {
```

Comment:
```suggestion
function get_prompt_from_diffusers_model(model: ModelData): string | undefined {
```

since the value `model.widgetData?.[0].text` is not guranteed to be there all the time